### PR TITLE
Refactoring the readme of the logging module

### PR DESCRIPTION
**Pull request by nandajavarma**

### CODE REVIEW

File: docs/COMPATIBILITY_MATRIX.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+## Compatibility
```
Comment (ralgozino) [inline docs/COMPATIBILITY_MATRIX.md:1]: This should be an `h1` header (a single `#`)

File: README.md [hunk 2]
```diff
@@ -1,85 +1,151 @@
...
+If you are new to KFD please refer to the [official documentation][kfd-docs] on how to get started with KFD.
 
+## Overview
 
-## Examples
+**Kubernetes Fury Logging** uses a collection of open source tools to provide the most resilient and robust logging stack for the cluster. 
 
-To see examples on how to customize Fury distribution packages with kustomize
-please go to [examples](examples).
+The central piece of the stack is the CNCF certified open source search engine [elasticsearch][elastic-search-page], combined with its the analytics and visualization platform [kibana][kibana-page]. The logs are collected using a node-level data collection and enrichment agent [fluentbit][fluentbit-page](deployed as `Daemonsets`), pushing it to the Elasticsearch via [fluentd][fluentd-page].
```
Comment (ralgozino) [inline README.md:20]: I think something's missing between `with` and `its` here: 
>[...]combined with its the analytics[...]

File: README.md [hunk 3]
```diff
@@ -1,85 +1,151 @@
...
 
-- [Kubernetes](https://kubernetes.io) >= `v1.18.0`
-- [Furyctl](https://github.com/sighup-io/furyctl) package manager to install Fury packages
-- [Kustomize](https://github.com/kubernetes-sigs/kustomize) >= `v3`
+If you are new to KFD please refer to the [official documentation][kfd-docs] on how to get started with KFD.
 
+## Overview
 
-## Examples
+**Kubernetes Fury Logging** uses a collection of open source tools to provide the most resilient and robust logging stack for the cluster.
```
Comment (ralgozino) [inline README.md:18]: trailing white space here make the linter sad

**Review by ralgozino — CHANGES_REQUESTED** [review]
looking good! just some minor adjustments

File: docs/COMPATIBILITY_MATRIX.md (hunk 1 above)
Comment (nandajavarma) [reply docs/COMPATIBILITY_MATRIX.md]: done!

File: README.md (hunk 3 above)
Comment (nandajavarma) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (nandajavarma) [reply README.md]: aaand done

**Review by ralgozino — APPROVED** [review]

**Pull request merged**